save-agent/src/linuxX64Main/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -141,7 +141,7 @@ class SaveAgent(private val config: AgentConfiguration,
     }
 
     private fun runSave(cliArgs: String): ExecutionResult = ProcessBuilder(true, FileSystem.SYSTEM)
-        .exec(config.cliCommand.let { if (cliArgs.isNotEmpty()) "$it $cliArgs" else it }, logFilePath.toPath())
+        .exec(config.cliCommand.let { if (cliArgs.isNotEmpty()) "$it $cliArgs" else it }, ".", logFilePath.toPath())
```

Comment:
May be we can put just `""`? In this case Pb won't change directory at all, of course 'changing dir to current' does not give a large load, however it's still an extra action

Reply:
OK, changed